packages/react-heartwood-components/src/components/BigCalendar/BigCalendar.js

```diff
@@ -77,7 +77,7 @@ class BigCalendar extends Component<Props, State> {
 		doubleClickTime: 250,
 		doubleClickToCreate: false, // defaults to single click
 		eventTimeFormat: 'h:mma',
-		headerCellDowFormat: 'dd',
+		headerCellDowFormat: 'ddd',
```

Comment:
This is a residual fix for SBL-1495 that probably isn't worth it's own PR if we can help it - sneaking it in here